cpp/ps0001/ps0001_sol.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+int main()
+{
+    // prompting user for 3 numbers to add 1.5/2
+    std::cout<<"Hello! please enter the numbers\n";
+    unsigned short a,b,c,sum;
+    std::cin>>a>>b>>c;
+    
+    // outputting the result of the summation process 3/3
+    sum=(a+b+c);
+    std::cout<<"the sum is: "<<sum<<"\n";
```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mar as your program is user-friendly ⭐